src/mappings/dayUpdates.ts

```diff
@@ -4,33 +4,31 @@ import { BigInt, BigDecimal, EthereumEvent } from '@graphprotocol/graph-ts'
 import { Pair, Bundle, Token, UniswapFactory, UniswapDayData, PairDayData, TokenDayData } from '../types/schema'
 import { ONE_BI, ZERO_BD, ZERO_BI, FACTORY_ADDRESS } from './helpers'
 
-export function updateUniswapDayData(event: EthereumEvent): void {
+export function updateUniswapDayData(event: EthereumEvent): UniswapDayData {
   let uniswap = UniswapFactory.load(FACTORY_ADDRESS)
   let timestamp = event.block.timestamp.toI32()
   let dayID = timestamp / 86400
   let dayStartTimestamp = dayID * 86400
   let uniswapDayData = UniswapDayData.load(dayID.toString())
-  if (uniswapDayData == null) {
-    let uniswapDayData = new UniswapDayData(dayID.toString())
+  if (uniswapDayData === null) {
+    uniswapDayData = new UniswapDayData(dayID.toString())
     uniswapDayData.date = dayStartTimestamp
     uniswapDayData.dailyVolumeUSD = ZERO_BD
     uniswapDayData.dailyVolumeETH = ZERO_BD
     uniswapDayData.totalVolumeUSD = ZERO_BD
     uniswapDayData.totalVolumeETH = ZERO_BD
     uniswapDayData.dailyVolumeUntracked = ZERO_BD
-    uniswapDayData.totalLiquidityUSD = ZERO_BD
-    uniswapDayData.totalLiquidityETH = ZERO_BD
-    uniswapDayData.txCount = ZERO_BI
-    uniswapDayData.save()
   }
-  uniswapDayData = UniswapDayData.load(dayID.toString())
```

Comment:
This pattern is repeated a few times where 

1. the code attempts to load the entity
2. if it doesn't exist, it's created but with a new scoped `let`
3. it's saved (if it didn't exist)
4. it's loaded again (redundant either way)
5. it's saved again.

Instead, this can be build with a single load & save.